lxd/storage_volumes.go

```diff
@@ -294,7 +294,7 @@ func storagePoolVolumesTypePost(d *Daemon, r *http.Request) response.Response {
 
 	// If we're getting binary content, process separately.
 	if r.Header.Get("Content-Type") == "application/octet-stream" {
-		return createStoragePoolVolumeFromBackup(d, projectName, poolName, r.Body)
+		return createStoragePoolVolumeFromBackup(d, projectName, r.Body, poolName, r.Header.Get("X-LXD-name"))
```

Comment:
@stgraber I've not switched this route to use `X-LXD-pool` header for pool name, as it was previously (even before the `createStoragePoolVolumeFromBackup` function was added) using the pool name encoded into the last part of the URL path, see https://github.com/lxc/lxd/commit/049401097ed4f20a824d8e6a5293e42084e15564#diff-b53e8affc6d5826a7bac3c3ea902581bL322

So I couldn't bring this API route totally in line with how instance imports are structured.

Reply:
Makes sense since unlikely instances that are a top-level collection, volumes are a per-pool collection.